blocks/cards/cards.css

```diff
@@ -203,10 +202,46 @@ main .nolink .cards > ul > li {
   text-align: left;
 }
 
+main .nolink .cards > ul > li .cards-card-image {
+  height: 135px;
+  width: 240px;
+  display: -ms-flexbox;
+  display: flex;
+  -ms-flex-item-align: start;
```

Comment:
Do we need the non-standard CSS properties? I'd not use them unless there's no support for standard ones in modern browsers.  You can use https://caniuse.com/ to check browser support.

Reply:
Removed